release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -106,6 +107,33 @@ the original scheduling decisions are no longer appropriate for certain nodes.
 See xref:../nodes/scheduling/nodes-descheduler.adoc#nodes-descheduler[Evicting Pods using the descheduler]
 for more information.
 
+[id="ocp-4-4-logging"]
+=== Logging
+
+[id="ocp-4-4-sending-cluster-logs-using-fluent-syslog-plug-in"]
+==== Sending cluster logs using the Fluentd syslog plug-in (RFC 3164)
+
+Due to changes introduced with the Log Forwarding feature in {product-title}
+4.3, you could no longer use the Fluentd syslog plug-in to forward logs to an
+external syslog server. In {product-title} 4.4, this functionality is restored
+and you can use the syslog plug-in. Note that the procedure for configuring the
+plug-in has changed. For more information, see
+xref:../logging/config/cluster-logging-external.adoc#cluster-logging-collector-syslog_cluster-logging-external[Sending logs using the Fluentd syslog plug-in (RFC 3164)].
+
+[id="ocp-4-4-networking"]
+=== Networking
+
+[id="ocp-4-4-sctp-on-ocp"]
+==== Stream Control Transmission Protocol (SCTP) on {product-title}
+
+If your cluster is installed on bare metal infrastructure, you can enable SCTP
```

Comment:
Not sure if I should peer review my own text, but maybe this instead:

> SCTP is a reliable message based protocol that runs on top of an IP network.
> If your cluster is installed on bare metal infrastructure, you can enable SCTP on the hosts in the cluster.
> When enabled, you can use SCTP as a protocol with both Pods and Services.

Reply:
Haha! I'll go with your new text.